flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -671,8 +671,11 @@ static void go_enable_fly_home()
 		vtol_fsm_path_desired.End[2] = -RTH_MIN_ALTITUDE;
 	}
 
-	vtol_fsm_path_desired.StartingVelocity = RTH_VELOCITY;
-	vtol_fsm_path_desired.EndingVelocity = RTH_VELOCITY;
+	float rth_vel = 3.0f;
```

Comment:
That makes me really nervous.  Auditing the Get /  UAVObjGetInstanceDataField appears to show paths where these functions can fail, in which case rth_vel is left set to an undefined value.  Granted this is unexpected on the path follower settings object.

In accordance with #1792, we could alter those paths to fault/reset instead, and it'd start to be safe..